securedrop/journalist_templates/col.html

```diff
@@ -141,12 +134,21 @@ <h3>{{ gettext('Reply') }}</h3>
     {% with %}
       {% set modal_data = {
                             "modal_id": "delete-collection-confirmation-modal",
-                            "modal_header": gettext('Delete Confirmation'),
-                            "modal_body": gettext('Are you sure you want to delete this collection?'),
+                            "modal_header": gettext('Are you sure?'),
+                            "modal_body": gettext('<p>
+Deleting the account for <b>{source}</b> will:
+<ul>
+<li>prevent them from logging in with their codename again
+<li>prevent your organization from sending replies
+</ul>
+</p>
+<p>
+All files, messages, and replies will also be deleted when their account is deleted.
+</p>'.format(source=source.journalist_designation)),
                             "cancel_id": "cancel-collection-deletions",
                             "submit_id": "delete-collection-button",
                             "submit_btn_type": "danger",
-                            "submit_btn_text": gettext('Delete')
+                            "submit_btn_text": gettext('Yes, Delete Source Account')
```

Comment:
Same height discrepancy between these two buttons.

Reply:
:heavy_check_mark: cut the knot and just wrapped them in a flexbox